addons/sale/views/sale_order_views.xml

```diff
@@ -261,13 +261,15 @@
                     type="object"
                     data-hotkey="shift+g"
                     class="oe_highlight"
+                    groups='account.group_account_user'
```

Comment:
Whoever has write access to the SO is expected to be able to capture/void the transaction(s) of the SO, not only accountants

Reply:
My bad, we should use the `group_sale_salesman` and not the `group_account_invoice` everywhere in sale.
We give this group rights on transactions in `sale/security/ir_rules.xml`, through `payment_transaction_salesman_rule`.

Reply:
Well, they should only have access to the transactions linked to the orders they can access ideally.
Is there another way to bypass the m2m than `auto_join` ?